src/azul/service/manifest_controller.py

```diff
@@ -231,8 +231,12 @@ def get_manifest_async(self,
             status = body.pop('Status')
             command_line: FlatJSON = body.pop('CommandLine', None)
             headers = {k: str(v) for k, v in body.items()}
-            new_body = None if command_line is None else ''.join(
-                f'\nDownload the manifest in {shell} with `curl` using:\n\n{cmd}\n'
-                for shell, cmd in command_line.items()
-            )
+            if command_line is None:
+                new_body = None
+            else:
+                headers['Content-Type'] = 'text/plain'
```

Comment:
While it may make sense to set the content type explicitly, I want to understand why a default of JSON was used when the body was a string.

Reply:
If no `Content-Type` header is specified in the response, a chalice specifies `application/json` by default.

https://aws.github.io/chalice/tutorials/basicrestapi.html#customizing-the-http-response

> Customizing the HTTP Response
> …
> Specify response bodies that are not `application/json`

https://aws.github.io/chalice/topics/views.html#view-function-return-values

> View Function Return Values
> …
> * A bytes type response body must have a Content-Type header value that is present in the app.api.binary_types list in order to be handled properly.
> * Any other return value will be serialized as JSON and sent back as the response body with content type application/json.

https://github.com/aws/chalice/blob/670130e47078308eac7d316259e7074203c6b659/chalice/app.py#L533

```
                # If a user does not provide a content type header, which can
                # happen if they return a python type instead of a ``Response``
                # type, then we assume the content is application/json.
```

https://github.com/aws/chalice/blob/670130e47078308eac7d316259e7074203c6b659/chalice/app.py#L1191

```
            'content_types': actual_kwargs.pop('content_types',
                                               ['application/json']),
```